utils/rpms/daos.spec

```diff
@@ -393,6 +398,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r daos_agent
 %{_libdir}/*.a
 
 %changelog
+* Tue Oct 22 2020 Maureen Jean <maureen.jean@intel.com> - 1.1.1-4
+- add fault_status to daos-tests files list
```

Comment:
Expand on the log entry here to add the details about building with[out] faults by use of `%{build_type}`.